install_wordpress_edition.sh

```diff
@@ -61,6 +61,12 @@ echo
 
 echo "Starting Plesk Installation"
 ./plesk-installer install plesk --preset Recommended --with fail2ban modsecurity spamassassin mailman psa-firewall pmm health-monitor
+OUT=$?
+if [ $OUT -ne 0 ];then
+  echo
+  echo "Plesk Installation Failed. Plese see logged lines above for error handling"
```

Comment:
Please use: "An error occurred! The installation of Plesk failed. Please see logged lines above for error handling!"

Reply:
Done